client/my-sites/people/followers-list/index.jsx

```diff
@@ -16,290 +16,286 @@ import PeopleListItem from 'calypso/my-sites/people/people-list-item';
+	getTotalFollowersForQuery,
+} from 'calypso/state/followers/selectors';
+import { removeFollower } from 'calypso/state/followers/thunks';
 
 /**
  * Stylesheet dependencies
  */
 import './style.scss';
+import { connect } from 'react-redux';
 
-const maxFollowers = 1000;
+const MAX_FOLLOWERS = 1000;
 
-const Followers = localize(
-	class FollowersComponent extends Component {
-		constructor() {
-			super();
+class Followers extends Component {
+	constructor() {
+		super();
 
-			this.infiniteList = React.createRef();
-		}
+		this.infiniteList = React.createRef();
+	}
 
-		state = {
-			bulkEditing: false,
-		};
+	state = {
+		bulkEditing: false,
+	};
 
-		renderPlaceholders() {
-			return <PeopleListItem key="people-list-item-placeholder" />;
-		}
+	renderPlaceholders() {
+		return <PeopleListItem key="people-list-item-placeholder" />;
+	}
 
-		fetchNextPage = () => {
-			const actions = 'email' === this.props.type ? EmailFollowersActions : FollowersActions;
-			const store = 'email' === this.props.type ? EmailFollowersStore : FollowersStore;
-			const paginationData = store.getPaginationData( this.props.fetchOptions );
-			const analyticsAction =
-				'email' === this.props.type
-					? 'Fetched more email followers with infinite list'
-					: 'Fetched more followers with infinite list';
-
-			let page = this.props.currentPage + 1;
-			if ( paginationData && paginationData.followersCurrentPage ) {
-				page = paginationData.followersCurrentPage + 1;
-			}
+	fetchNextPage = () => {
+		const analyticsAction =
+			'email' === this.props.type
+				? 'Fetched more email followers with infinite list'
+				: 'Fetched more followers with infinite list';
 
-			actions.fetchFollowers( Object.assign( this.props.fetchOptions, { page } ) );
-			gaRecordEvent( 'People', analyticsAction, 'page', page );
-		};
-
-		removeFollower( follower ) {
-			const listType = 'email' === this.props.type ? 'Email Follower' : 'Follower';
-			gaRecordEvent( 'People', 'Clicked Remove Follower Button On' + listType + ' list' );
-			accept(
-				<div>
-					<p>
-						{ this.props.translate(
-							'Removing followers makes them stop receiving updates from your site. If they choose to, they can still visit your site, and follow it again.'
-						) }
-					</p>
-				</div>,
-				( accepted ) => {
-					if ( accepted ) {
-						gaRecordEvent(
-							'People',
-							'Clicked Remove Button In Remove ' + listType + ' Confirmation'
-						);
-						( 'email' === this.props.type
-							? EmailFollowersActions
-							: FollowersActions
-						).removeFollower( this.props.site.ID, follower );
-					} else {
-						gaRecordEvent(
-							'People',
-							'Clicked Cancel Button In Remove ' + listType + ' Confirmation'
-						);
-					}
-				},
-				this.props.translate( 'Remove', { context: 'Confirm Remove follower button text.' } )
-			);
-		}
+		this.props.incrementPage();
+		gaRecordEvent( 'People', analyticsAction, 'page', this.props.currentPage + 1 );
```

Comment:
Would be cool if we could port those to Redux, too. Not necessarily in this PR, of course.

Reply:
I'll follow up with another PR 👍🏻